### ISSUE TITLE
Placeholders in the configuration URLs

### ISSUE THREAD

**Issue by ancorgs** [jangouts/jangouts#140]
Some configuration parameters like "janusServer", "janusServerSSL" or "httpsUrl" are URLs. In many situations it would be useful to accept placeholders as part of those urls, specially the host name, so it's possible to specify something like these in servers replying to different hostnames.

``` json
{
  "janusServer": "ws://%{hostname}/janus",
  "janusServerSSL": "wss://%{hostname}/janus"
}
```

The placeholders would be substituted by the corresponding part of the current URL the browser is using to access Jangouts.

**Comment by mohsini172** [comment]
Can I add this using gulp?

**Comment by mohsini172** [comment]
I have modified the gulp serve task and implemented this functionality. I need some guide if I can use the gulp. If yes how the parameter will be passed? Do I need to edit the other tasks for this?

**Comment by imobachgs** [comment]
@mohsini172 Sorry, but the idea is not to use gulp, but to replace those placeholders with information from the URL on runtime. For instance, given the example configuration in this issue, if you're connecting to Jangouts using an URL `https://myserver.com`, then `janusServer` value should be `ws://myserver.com/janus` and it will happen automatically.

Don't hesitate to ask if it's still not clear.

**Comment by mohsini172** [comment]
I'm little confused what exactly issue is? Can you give a practical example to explain this. I mean what will be the initial configuration file will be look like and when run the server from which point in the code should I replace those values?

**Comment by imobachgs** [comment]
@mohsini172 Sure! Given a configuration file with these values:

```json
{
  "janusServer": "ws://%{hostname}/janus",
  "janusServerSSL": "wss://%{hostname}/janus"
}
```

If you connect your browser to `http://myserver.com/`, then `janusServer` should be interpreted as `ws://myserver.com/janus`. Of course, you don't need to change the configuration file itself, but just to modify the configuration on runtime. The benefit is that you won't need to adjust the configuration file by hand to match the hostname.

Is it more clear now?

Thanks!

**Comment by mohsini172** [comment]
Perfectly understand. Just one more thing from which file can I start with. I mean in which file the configuration is reading.

**Comment by imobachgs** [comment]
Closed by #177.
